### PR TITLE
Setting up Django dev env & Project folders

### LINKED ISSUE #4: Setup Django
In order to develop new features, we first need to setup the django framework in the project as well as integrate bringing up django in the bootstrap.sh

### PR DESCRIPTION
**Pull request by Gal-f**
- Bootstrapped the project and app foundations to be filled with content.
The new 'StuDash' and 'StuDash_home' folders will contain the homepage material.

- Added a command to start the Django app when starting the VM.

- Bug fix - Minor adjustments to port forwarding.

This PR resolves #4 .

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -55,7 +55,7 @@ Vagrant.configure("2") do |config|
   #   vb.gui = true
   #
   #   # Customize the amount of memory on the VM:
-  vb.memory = "1024"
+    vb.memory = "1024"
```
Comment (ravidbro) [inline Vagrantfile:58]: Is that intentional?
The indentation looks different from the other configs

Comment (Gal-f) [reply Vagrantfile:58]: Yes, following @ifireball 's comment on this on the previous PR.

Comment (ravidbro) [reply Vagrantfile:58]: OK.
Looks fine to me (I also cloned it and ran it on my machine, looks great).
@ifireball do you have any comments, or can we merge it?

**Review by ifireball — APPROVED** [review]
Using capital letters in a python module name is uncommon, but it'll probably not cause you too much trouble down the road.

Otherwise, LGTM.

**Comment by ravidbro** [comment]
@Gal-f do you want to change the capital letters, or should we merge it like that?

**Comment by Gal-f** [comment]
I'll keep that in mind, thanks for the advice!
Please merge, others' work depends on it so I'd prefer to close this asap.

**Comment by ravidbro** [comment]
I'm merging it,
You can fix it in a different PR

**Pull request merged**